common/scala/src/main/scala/org/apache/openwhisk/core/connector/Message.scala

```diff
@@ -256,6 +286,7 @@ case class Metric(metricName: String, metricValue: Long) extends EventMessageBod
 }
 
 object Metric extends DefaultJsonProtocol {
+  val typeName = "Metric"
```

Comment:
Probably not used. We have `typeName` in class also

Reply:
I think it's still being used here: https://github.com/apache/openwhisk/blob/f04275a825346da52d8616cf241f7a6ae78dccbc/core/monitoring/user-events/src/main/scala/org/apache/openwhisk/core/monitoring/metrics/EventConsumer.scala#L99